pkg/util/iptables/iptables_unsupported_test.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2021 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package iptables
+
+import (
+	"testing"
+)
+
+func Test_grabIptablesFileLock(t *testing.T) {
```

Comment:
why add `iptables_unsupported_test.go` file?

Reply:
in linux is ok, in Mac  make lint is error ,because the func not used. we can ignore it.

Reply:
ok, it sounds reasonable to add this file.